src/utils/intl.ts

```diff
@@ -0,0 +1,14 @@
+import { createIntl, createIntlCache } from '@formatjs/intl';
+
+// https://formatjs.io/docs/intl
+
+const cache = createIntlCache();
+
+// For use outside of React components
+export const intl = createIntl(
```

Comment:
@rdig Just want to check you had something like this in mind? https://github.com/JoinColony/colonyCDapp/pull/83/commits/79ec4a0882be12fe19aa3d668dd6cfc692e66fb5#diff-151319ce8167c54c18d1d2f343ba2df500e65b24ee675ba59b57318f31fdd4fb

I.e a function you can pass messages to that will return you a new intl object each time it's called? Instead of adding the various sets of messages to the intl obj that I had previously created in `utils/intl`?